binary/relations.proto

```diff
@@ -5,44 +5,47 @@ package io.substrait;
 import "type.proto";
 import "expression.proto";
 import "selection.proto";
+import "extensions.proto";
+import "google/protobuf/any.proto";
 
 option java_multiple_files = true;
 option csharp_namespace = "Substrait.Protobuf";
 
 message RelCommon {
 
-    oneof kind {
+    oneof emit_kind {
         Direct direct = 1;
         Emit emit = 2;
     }
 
     Hint hint = 3;
-    RuntimeConstraint constraint = 4;
+    io.substrait.extensions.AdvancedExtension advanced_extension = 4;
 
     message Direct {}
     message Emit {
-        repeated int32 output_mapping = 1;
+        repeated int32 output_mapping = 1 [packed = true];
```

Comment:
This is already packed by default since we're using `proto3`: https://developers.google.com/protocol-buffers/docs/encoding#packed.

Reply:
fixed (in the separated out patch)